### PR TITLE
Add an example for conditional probability

### PR DESCRIPTION
**Pull request by user202729**
I finally got around to learn this after _some stuff_ happens… probability is actually quite Olympiad-like compared to, say, algebraic geometry.

(and realize in the meantime that it might make sense for the sake of being consistent to define measurable function to lift measurable to measurable, if you don't have a topology on the codomain. But why would there be such a case?)

### CODE REVIEW

File: tex/measure/martingale.tex [hunk 1]
```diff
@@ -192,11 +211,57 @@ \section{Conditional expectation}
...
 The way that this is typically handled in measure theory
 is to use the notion of sub-$\sigma$-algebra that we defined.
-Let $\SF$ be a sub-$\sigma$-algebra which captures the information
-This means we create a function assigning the
-``conditional expectation'' to \emph{every} point $\omega \in \Omega$,
-which is measurable with respect to $\SF$.
-\todo{give the example}
+
+But first, we should explain what $\EE(X \mid X+Y)$ means first --- why are we conditioning on
+another random variable instead of an event?
```
Comment (user202729) [inline tex/measure/martingale.tex:216]: Is this "common high school knowledge"? (I sort of didn't know this before)

File: tex/measure/martingale.tex [hunk 2]
```diff
@@ -192,11 +211,57 @@ \section{Conditional expectation}
...
+
+To motivate conditioning on a random variable, consider the following situation.
```
Comment (user202729) [inline tex/measure/martingale.tex:218]: (better example is welcome. I'm uncreative 😢)

File: tex/measure/martingale.tex [hunk 3]
```diff
@@ -192,11 +211,57 @@ \section{Conditional expectation}
...
+Suppose that the weather tomorrow depends on the weather today and the random fluctuations.
+So we may have statements such as:
+\begin{align*}
+	\Pr(\text{it rains tomorrow}\mid \text{it rains today}) &= 0.6, \\
```
Comment (user202729) [inline tex/measure/martingale.tex:222]: We actually never use `\Pr` anywhere else in the document. Do you prefer `\mu` instead? (Does `\mu(A \mid B)` look weird?)

File: tex/measure/martingale.tex (hunk 1 above)
Comment (vEnhance) [reply tex/measure/martingale.tex:216]: I don't think so.

**Comment by vEnhance** [comment]
Thanks, this is fine. (I don't know the etymology of "martingale" off my head.)

**Comment by user202729** [comment]
By the way what are the statements of these two problems? 

image

**Comment by vEnhance** [comment]
ABRACADABRA is https://www.jeremykun.com/2014/03/03/martingales-and-the-optional-stopping-theorem/

USA TST 2018/2 is https://aops.com/community/p9513099

**Pull request merged**